packages/bot/src/scratch/index.js

```diff
@@ -1,28 +1,29 @@
 import './blocks';
 import './hooks';
 import {
-    isMainBlock,
     save,
-    disable,
     addLoadersFirst,
     cleanUpOnLoad,
     addDomAsBlock,
-    backwardCompatibility,
-    fixCollapsedBlocks,
-    fixArgumentAttribute,
-    removeUnavailableMarkets,
-    strategyHasValidTradeTypeCategory,
     cleanBeforeExport,
-}                                         from './utils';
-import { showDialog }                     from '../services/tradeEngine/utils/helpers';
-import Interpreter                        from '../services/tradeEngine/utils/interpreter';
-import GTM                                from '../utils/gtm';
-import createError                        from '../utils/error';
-import { translate, xml as translateXml } from '../utils/lang/i18n';
-import { getLanguage }                    from '../utils/lang/lang';
-import { observer as globalObserver }     from '../utils/observer';
-
-export const scratchWorkspaceInit = async (scratch_area_name, scratch_div_name) => {
+}                                     from './utils';
+import ActiveSymbols                  from './utils/active-symbols';
+import ContractsFor                   from './utils/contracts-for';
+import ServerTime                     from './utils/server-time';
+import TradingTimes                   from './utils/trading-times';
+import Interpreter                    from '../services/tradeEngine/utils/interpreter';
+import createError                    from '../utils/error';
+import { translate }                  from '../utils/lang/i18n';
+import { observer as globalObserver } from '../utils/observer';
+
+export const scratchWorkspaceInit = async (scratch_area_name, scratch_div_name, root_store) => {
+    const server_time    = new ServerTime(root_store.ws);
```

Comment:
Do we really need to pass `scratch_area_name` & `scratch_div_name` from `app.jsx`?
we can just hardcode `scratch_area` & `scratch_div` at line@28,29